### ISSUE TITLE
Accompagnement DINUM/Etalab sur la license

### ISSUE THREAD

**Issue by bzg** [LAB-MI/attestation-deplacement-derogatoire-q4-2020#20]
Si besoin, je suis à disposition pour aider sur les choix de licence ou sur les questions d'ouverture en général.

**Comment by rgrunbla** [comment]
Bonjour,

La licence s'applique-t-elle bien sur l'ensemble des fichiers présents dans ce dépôt ? Je suis d'avis que oui, puisqu'il n'y a pas d'exceptions sur les fichiers présents dans le texte de la licence (je pense aux typographies, ainsi qu'aux fichiers svg qui sont par ailleurs à la fois des ressources graphiques et des codes sources).

Merci :)

ps: j'écris ici car cela me semble être l'endroit le plus adapté pour poser la question

**Comment by bzg** [comment]
Bonjour, oui par défaut la licence s'applique à l'ensemble des fichiers.

Comme il y a parmi eux les fichiers du Marianne©, il faudrait préciser que ceux-ci ne sont pas couverts par la licence.

Voir le court paragraphe que j'ai ajouté à ce sujet [dans ce dépôt](https://github.com/etalab/bulma-theme-gouv-fr#licence).

**Comment by rgrunbla** [comment]
Cependant, le fait que Marianne soit protégée par le droit d’auteur, ou même qu'elle soit déposée à l'INPI (par exemple) ne veut pas dire qu'elle ne peut pas être diffusée sous licence MIT (la licence MIT protège d'ailleurs elle aussi le droit d'auteur, enfin certains). J'ai l'impression qu'il s'agit de deux questions plutôt orthogonales, et donc il n'est pas si clair que la licence n'est pas _effectivement_ diffusée sous licence MIT.

**Comment by bzg** [comment]
Non, on ne peut pas soi-même publier sous licence MIT (ou toute autre licence libre) une oeuvre dont les droits sont détenus par un tiers.

**Comment by rgrunbla** [comment]
Wikimedia diffuse son logo sous une licence libre (exemple: https://commons.wikimedia.org/wiki/File:Wikipedia-logo-v2.svg, sous une licence creative commons), et il est rediffusé par des tiers, alors que ce logo est bien déposé à l'INPI. Est-ce que cela veut dire que c'est illégal ?

Doit ont donc considérer ce dépôt comme illégal, puisqu'il rediffuse la police d'écriture Marianne sous licence MIT?

Notons que le © de la page du gouvernement sur la typographie n'a pas de portée juridique en France: https://www.inpi.fr/fr/faq/l-utilisation-du-sigle-c-copyright-est-elle-soumise-autorisation

L'état ne possède pas les droits de la police ?

**Comment by bzg** [comment]
> Wikimedia diffuse son logo sous une licence libre (exemple: https://commons.wikimedia.org/wiki/File:Wikipedia-logo-v2.svg, sous une licence creative commons), et il est rediffusé par des tiers, alors que ce logo est bien déposé à l'INPI. Est-ce que cela veut dire que c'est illégal ?

Il s'agit du logo de Wikipédia, l'un des projets du mouvement Wikimedia. L'usage de ce logo est bien encadré, voir ce qui est écrit dans [cette page](https://commons.wikimedia.org/wiki/File:Wikipedia-logo.png):

> ™ Wikimedia Foundation, Inc.
This file is (or includes) one of the official logos or designs used by the Wikimedia Foundation or by one of its projects. Use of the Wikimedia logos and trademarks is subject to the Wikimedia trademark policy and visual identity guidelines, and may require permission.

Non, l'Etat ne possède pas les droits de la police Marianne.

Notez que je n'aurai peut-être pas le temps, par la suite, de répondre à d'autres questions que celles des contributeurs de ce dépôt.

**Comment by rgrunbla** [comment]
Merci pour les précisions.

Le fait que l'usage du logo soit encadré ne l'empêche pas d'être pour autant diffusé sous une licence libre:

>Permission is granted to copy, distribute and/or modify this document under the terms of the GNU Free Documentation License, Version 1.2 or any later version published by the Free Software Foundation; with no Invariant Sections, no Front-Cover Texts, and no Back-Cover Texts. A copy of the license is included in the section entitled GNU Free Documentation License. 

Que l'usage de la typographie Marianne soit encadré ne veut donc pas dire qu'elle ne peut pas être diffusée sous licence libre. La question reste donc entière, et les explications apportées ici sont donc peu satisfaisantes.

Si l'état ne dispose pas des droits sur la police Marianne, ce dépôt possède donc une licence inadaptée, et les contributrices et contributeurs ont donc été trompés en pensant à contribuer à un dépôt sous licence MIT qui ne l'est donc pas. @LAB-MI, ça ne fait pas très très sérieux. Surtout qu'il serait possible de remplacer la typographie par une typographie Spectral.

ps: la documentation, le licensing, ça fait partie intégrante du dépôt, en discuter le bienfondé est donc contribuer au dépôt

**Comment by bzg** [comment]
> Que l'usage de la typographie Marianne soit encadré ne veut donc pas dire qu'elle ne peut pas être diffusée sous licence libre.

Vous faites erreur.

Le logo Wikipédia est diffusé sous licence libre par son auteur, qui possède aussi la marque liée. La police Marianne ne peut être placée sous licence libre que par son auteur, pas par un tiers.

C'est ce qui fait que la licence MIT de ce dépôt ne peut pas s'appliquer à la police.

Je pense que le @LAB-MI n'a pas voulu induire qui que ce soit en erreur et que la licence MIT de ce dépôt est tout à fait correcte pour tous les fichiers, hormis ceux de la police. Aucune contribution extérieure n'a porté sur ces fichiers, donc aucun contributeur n'a été induit en erreur.

Discuter d'un projet, dans une issue, une PR, sur Twitter ou seul dans sa salle de bain est effectivement une forme de contribution (toute relative dans le cas de la salle de bain), mais seules les contributions sous forme de modifications des fichiers couverts par la licence constitue une "contribution" au sens que celle-ci donne à ce terme.

**Comment by rgrunbla** [comment]
>C'est ce qui fait que la licence MIT de ce dépôt ne peut pas s'appliquer à la police.

Sauf si la police Marianne a été placée sous licence libre par son auteur, et que ce dépôt ne fait que réutiliser cette police, avec la licence associée, respectant ainsi le choix de l'auteur… 

Si maintenant, on me dit que la personne à l'origine de la police Marianne ne l'a pas placée sous licence libre, et donc que cette licence est l'initiative de personnes qui ne sont pas à l'origine de cette police, sans autorisation, alors il serait vraiment temps de changer la licence du dépôt. Est-le cas ?

**Comment by bzg** [comment]
La police Marianne n'a pas été placée sous licence libre par son auteur.

Il n'est pas besoin de changer la licence de ce dépôt, juste d'indiquer qu'elle ne couvre pas les fichiers de la police Marianne, comme je le fais [dans le README de ce dépôt](https://github.com/etalab/bulma-theme-gouv-fr#licence), déjà cité plus haut.

Au revoir !
